onboarding_documentation/technical_documentation/analyst_tutorial/analyst_tutorial.md

```diff
@@ -0,0 +1,122 @@
+
+In this tutorial, we will begin with raw fastq files and process them using the large-cohort pipeline. This pipeline performs several steps:
+
+1. Aligns the reads to the reference genome.
+2. Conducts quality control checks.
+3. Calls variants and combines these variant calls.
+4. Performs an ancestry analysis.
+5. Plots the results of the ancestry analysis.
+
+While the large-cohort pipeline includes additional stages, for the purpose of this tutorial, we will only execute up to and including the Ancestry stage, which carries out Principal Component Analysis (PCA).
+
+For a description of what each stage of the `large-cohort` pipeline does, please see the `large-cohort` pipeline documentation [here](https://github.com/populationgenomics/production-pipelines#large-cohort-workflow)
+
+## Table of Contents
+- [1. Getting test data](#getting-test-data)
+- [2. Writing a config file](#writing-a-config-file)
+- [3. Submitting an analysis job](#submitting-an-analysis-job)
+- [4. Analysing the output](#analysing-the-output)
+
+## 1. Getting test data
+As an analyst it is vitally important we run our analysis jobs on test data before running them on real data. This is to ensure that the analysis job is working as expected, and that the output is as expected. This is also important to ensure that the analysis job is not going to take too long (and therefore cost too much!) to run on real data. As pipelines are constantly being updated, it is also important to run the analysis job on test data to ensure that the pipeline has not been broken by a recent update.
+
+For this tutorial we will be using the `bioheart` dataset, which is a collection of genomes from different sources. First things first, let's check if you have access to the `bioheart` dataset. To do this, run the following command:
+
+```bash
+gsutil ls gs://cpg-bioheart-main/
+```
+
+The output of the above should list a number of files ending in `.fastq.gz`. These are read files, which are the input to the pipeline. If you do not see these files, then you do not have access to the `bioheart` dataset, please contact your manager.
+
+### Subsetting the bucket
+Our aim is to validate the pipeline's functionality and output by running it on a subset of the bioheart dataset. To do this we will use the script [create_test_subset.py](https://github.com/populationgenomics/metamist/blob/f6c226d08a8ee9875014d8c99cfe119742221efb/scripts/create_test_subset.py) in the metamist repository. This script selects a random set of samples from `bioheart` and copies their read files to the `bioheart-test` bucket. Execute the following command after navigating to the main branch of your metamist repository in the CLI.
+
+To run this script we must first be in the `metamist` repository. So navigate to your `metamist` repository, ensure you're on the `main` branch of `metamist` and then run the following command from CLI:
```

Comment:
It should be mentioned that gsutils needs to be installed, plus any other dependencies (analysis-runner, etc)

Reply:
You should mention that this script needs to be cloned if they haven't done so already